rust/azure_iot_operations_connector/Cargo.toml

```diff
@@ -25,6 +25,9 @@ notify-debouncer-full.workspace = true
 # tokio-stream = "0.1"
 thiserror.workspace = true
 log.workspace = true
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Maybe we should have this at the workspace level

Reply:
Agree, lets revisit when the scope is settled.